.github/workflows/build.yml

```diff
@@ -62,7 +62,7 @@ jobs:
       uses: microsoft/setup-msbuild@v1.1.3
     - name: Download Dalamud
       run: |
-        Invoke-WebRequest -Uri https://goatcorp.github.io/dalamud-distrib/latest.zip -OutFile latest.zip
+        Invoke-WebRequest -Uri https://goatcorp.github.io/dalamud-distrib/stg/latest.zip -OutFile latest.zip
```

Comment:
I'd rather wait until Dalamud is publicly released and _not_ change the Dalamud binary it's building against to the staging version, because staging can incorporate breaking changes and changes that are reverted before public release, and I develop against my actual game, which stays on public so that I can play.

Reply:
Completely fair. Once it got fully released i was going to swap this back. Thankfully, from what it looks like, it's a somewhat straightforward change.

Reply:
So, once dalamud gets a final release this can be swapped away from stg, but otherwise it won't build at all.  Might just set up a testing build thinger in a little bit.